clients/cloud/python/README.md

```diff
@@ -0,0 +1,55 @@
+# Overview
+
+Produce messages to and consume messages from Confluent Cloud using Confluent Python Client for Apache Kafka
```

Comment:
Since my inspiration for the project is this: https://www.rabbitmq.com/tutorials/tutorial-one-python.html
I'd love to see an explanation that places this example in the grand scheme of things and leaves place for other examples (many consumers for instance).

Reply:
I added a bit more into the README.  I figure we can expand on the explanations later as we flesh out the ideas with the blog posts.